### DISCUSSION TITLE
Delete message from usercard

### DISCUSSION THREAD

**Discussion by askepticaldreamer** [Chatterino/chatterino2#4383] [Ideas]
Someone requested the ability to delete messages from the usercard. I see there are already the other mod actions like ban/unban and timeouts on the usercard. Thoughts on adding the delete message button next to messages on the usercard?

**Comment by askepticaldreamer** [comment]
ahh, realized you can do this with show in message menu

**Comment by 8thony** [comment]
- Currently there is an open pull request for this feature. #5870
- Duplicate of #3069
